Towerfall/Assets/_Main/Scripts/Player/PlayerController.cs

```diff
@@ -8,16 +8,23 @@ namespace Towerfall.Controllers
     public interface IPlayerControllerInput
     {
         IObservable<float> JumpStart { get; }
-        IObservable<float> Move { get; }
+        IObservable<float> Run { get; }
         IObservable<Vector2> DashStart { get; }
     }
 
     public interface IPlayerControllerEvent
     {
+        void SetPlayerControllerData(IPlayerControllerData playerControllerData);
+
         IObservable<Unit> DashEnd { get; }
```

Comment:
This should be in `IPlayerControllerInput`

Reply:
c0da1b18286545cda7795908f1264d35c13d5a1f

Reply:
1637e433fda884bfa8b4a7739c561368b44cbd5a